api/api/src/test/scala/uk/ac/wellcome/platform/api/images/ImagesSimilarityTest.scala

```diff
@@ -0,0 +1,53 @@
+package uk.ac.wellcome.platform.api.images
+
+import uk.ac.wellcome.elasticsearch.ElasticConfig
+
+class ImagesSimilarityTest extends ApiImagesTestBase {
+
+  it(
+    "includes visually similar images on a single image if we pass ?include=visuallySimilar") {
+    withApi {
+      case (ElasticConfig(_, imagesIndex), routes) =>
+        val images = createVisuallySimilarImages(6)
+        val image = images.head
+        insertImagesIntoElasticsearch(imagesIndex, images: _*)
+        assertJsonResponse(
+          routes,
+          s"/$apiPrefix/images/${images.head.id.canonicalId}?include=visuallySimilar",
+          unordered = true) {
+          Status.OK ->
+            s"""
+           |{
+           |  $singleImageResult,
+           |  "id": "${image.id.canonicalId}",
+           |  "locations": [${digitalLocation(image.location)}],
+           |  "visuallySimilar": [
+           |    ${images.tail.map(imageResponse).mkString(",")}
+           |  ],
+           |  "source": {
+           |    "id": "${image.source.id.canonicalId}",
+           |    "type": "Work"
+           |  }
+           |}""".stripMargin
```

Comment:
bikeshed: funny indentation going on here imo. also i dont think we need the whole `stripMargin` thing, the string is never displayed in raw form as is parsed to a json obj and reformatted before being displayed in any test failures

Reply:
I agree but it's like this across like 10+ files so will leave as is for now

Reply:
Cool might do a whole clean up of the bikeshed PR at some point if its niggling me